controllers/categoriesController.js

```diff
@@ -0,0 +1,9 @@
+const { fetchCategories } = require("../models/categoriesModel.js");
+
+const getCategories = (request, response, next) => {
+  fetchCategories().then((categories) => {
+    response.status(200).send({ categories });
+  });
```

Comment:
ensure all model function invocations have a .catch block to make sure if for whatever reason the promise rejects it will be caught by your error handlers rather than not responding at all